plugins/rbac-backend/src/role-manager/role-manager.test.ts

```diff
@@ -658,7 +667,7 @@ describe('BackstageRoleManager', () => {
       );
       expect(result).toBeFalsy();
       expect(loggerMock.warn).toHaveBeenCalledWith(
-        'Detected cycle dependencies in the Group graph: [["group:default/team-a","group:default/team-b"]]. Admin/(catalog owner) have to fix it to make RBAC permission evaluation correct for group: group:default/team-a',
+        'Detected cycle dependencies in the Group graph: [["group:default/team-a","group:default/team-b"]]. Admin/(catalog owner) have to fix it to make RBAC permission evaluation correct for group: role:default/team-a',
```

Comment:
Error message looks strange, because message talks about group, but prints role.

Reply:
I think that's because you used: 

roleManager.hasLink(
   'user:default/mike',
   'role:default/team-b',
 ); 
      
 Second argument is role, but as far I remember casbin engine asks only about user or group entity reference like the second argument.

Reply:
It is kind of strange but hasLink will compare the user that we are checking permissions of against the roles that it is aware of. So the second argument will be a role so long as grouping policies are set up with roles. 

But I can update the error message to instead say: 'Detected cycle dependencies in the Group graph: [["group:default/team-a","group:default/team-b"]]. Admin/(catalog owner) have to fix it to make RBAC permission evaluation correct for role: role:default/team-a'

or: 'Detected cycle dependencies in the Group graph: [["group:default/team-a","group:default/team-b"]]. Admin/(catalog owner) have to fix it to make RBAC permission evaluation correct for groups: group:default/team-a, group:default/team-b'